### PR TITLE
Fix sym_lu

### PR DESCRIPTION
**Pull request by YingboMa**

### CODE REVIEW

File: src/solve.jl [hunk 1]
```diff
@@ -28,7 +28,7 @@ function sym_lu(A)
 
         for j = k+1:m
             L[j,k] = U[j, k] / U[k, k]
-            U[j,k:m] .= U[j,k:m] .- L[j,k] .* U[k,k:m]
+            U[j,k:m] .= U[j,k:m] .- Ref(L[j,k]) .* U[k,k:m]
```
Comment (ChrisRackauckas) [inline src/solve.jl:31]: why Ref?

Comment (YingboMa) [reply src/solve.jl:31]: So it behaves like a scalar in broadcasting. There's no guarantee that a symbolic object is a scalar.

Comment (ChrisRackauckas) [reply src/solve.jl:31]: tuple it?

Comment (ChrisRackauckas) [inline src/solve.jl:31]:
```suggestion
            U[j,k:m] .= U[j,k:m] .- (L[j,k],) .* U[k,k:m]
```

**Pull request merged**